src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -59,6 +61,9 @@
 
   protected abstract Map<TargetLanguage, Map<String, VersionBound>> protoPackageDependencies();
 
+  @Nullable
+  protected abstract Map<TargetLanguage, Map<String, VersionBound>> protoPackageTestDependencies();
```

Comment:
This is a protected method and the public accessor is `Map<String, VersionBound> protoPackageTestDependencies()`. Similar to `protoPackageDependencies` this method returns null if the language key is not found.